services/core-web/src/tests/components/noticeOfWork/applications/review/ReviewApplicationFeeContent.spec.js

```diff
@@ -31,292 +31,4 @@ describe("ReviewApplicationFeeContent", () => {
     const component = shallow(<ReviewApplicationFeeContent {...props} />);
     expect(component).toMatchSnapshot();
   });
-
-  it("MIN: isApplicationFeeValid === true if `adjustedTonnage` is set and exceeds proposed tonnage for MIN", () => {
```

Comment:
After converting the `ReviewApplicationFeeContent` component into a functional component, these tests no longer work as you can't spy on functions within a functional component.  As this type of testing is generally avoided in favour of user flow testing, I made the decision to get rid of them for now.  We have upcoming tickets to implement Cypress tests for high value areas, and I think NoW is definitely one of those.